### DISCUSSION TITLE
How to pass arrays from a response to a request downstream?

### DISCUSSION THREAD

**Discussion by dan-sproutward** [postmanlabs/postman-flows#243] [Q&A]
I can't figure this out. I have tried to stringify the array but config variable complains it recognizes the string as I list. I then stringified the contents of the array as a CSV string and add to the request with square brackets but config variable complains it recognizes the CSV as a list. I am stuck.

**Comment by areliBam** [comment]
What I'm doing at the moment, is selecting the list in the response, tagging on the "to-json" option, wrapping the entire selection in parenthesis and assigning it to a durable that gets passed on to the next request. This seems to be working fine. Hope it helps in your situation!

![image](https://user-images.githubusercontent.com/99707036/153993059-918ff1c1-2cff-4632-9a98-7bfba17aa92e.png)

**Comment by samay-v** [comment]
Hey @dan-sproutward 
You can assign the array/list directly to a variable in your next "Send request" block's modal 
![image](https://user-images.githubusercontent.com/28357106/154124101-f199f796-10f6-4565-b886-feab3ae3c619.png)

Make sure your request have examples defined so that flows can interpret the schema.
